### ISSUE TITLE
Latest version does not show flash messages (e.g. on error sending email)

### ISSUE THREAD

**Issue by sypets** [in2code-de/powermail#1087]
## reproduce

1.  added all TypoScript static includes
2. created a form
3. in Frontend send form
4. via Debugger in FormController:: line 385, set `$isSent` to false
5. `$this->addFlashMessage(` is then called, but no flash message displayed in frontend

```
$isSent = $mailPreflight->sendReceiverMail($mail, $hash);
if ($isSent === false) {
```
https://github.com/in2code-de/powermail/blob/b6e686506fcb815860dc305358057b07116d7351/Classes/Controller/FormController.php#L384

## reason

* the partial FlashMessages is called, but 
* the identifier of the FlashMessageQueue is set as (in ActionController):

`$identifier = 'extbase.flashmessages.' . $pluginNamespace;`

which evaluates to

`'extbase.flashmessages.tx_powermail_pi1'`

but the partial uses:

`<f:flashMessages queueIdentifier="powermail"/>`

If we change it to

`<f:flashMessages queueIdentifier="extbase.flashmessages.tx_powermail_pi1"/>`

the messages are displayed (but not styled).

## versions

* TYPO3: latest 12.4 (12.4 branch)
* powermail: 12.3.5 (rechecked with latest master/ main, has same problem)
* Composer installation: no
* PHP: 8.2

**Comment by mschwemer** [comment]
Thanks for your report.  The fix is available in the feature-branch and will be included in one of the next releases.

The styling is up to the fe developers.

If you need a styling, you can copy the alert classes from bootstrap in `Resources/Private/Build/Sass/_BootstrapExpanded.scss` to your stylesheet.

I will leave it open, until the release is out.

**Comment by mschwemer** [comment]
PR merged

**Comment by mschwemer** [comment]
Needed to revert the PR, because other flash messages were not shown and the form delivered a wrong result. 

See the behaviour tests without Typoscript

**Comment by mschwemer** [comment]
Finally solved with cb74a9c2224dc0405e3bc8ae36592002c44eef3b

**Comment by sypets** [comment]
@mschwemer Thank you!
